glide.yaml

```diff
@@ -39,6 +39,7 @@ import:
   version: 382f87b929b84ce13e9c8a375a4b217f224e6c65
 - package: github.com/xeipuuv/gojsonschema
 - package: github.com/blang/semver
+  version: 3.5.0
```

Comment:
@hferentschik I put fix version for the package which have releases and tags but some of package which we consume doesn't have tags/release in place :(